docs/source/guide/frontend.md

```diff
@@ -6,150 +6,282 @@ meta_title: Customize User Interface
 meta_description: Label Studio documentation for integrating the Label Studio frontend interface into your own machine learning or data labeling application workflow.
 ---
 
-The [Label Studio Frontend](https://github.com/heartexlabs/label-studio-frontend) (LSF) is the frontend library for Label Studio, based on React and mobx-state-tree and distributed as an NPM package. You can include it in your applications without using the Label Studio Backend (LSB) to provide data annotation support to your users. You can customize and extend the frontend library. 
+The [Label Studio Frontend](https://github.com/heartexlabs/label-studio-frontend) (LSF) is the main labeling interface distributed within Label Studio and as a separate package via NPM and Unpkg. You can integrate the LSF into your projects without Label Studio to provide data labeling capabilities to your users.
 
-LSF is located as a separate GitHub repository: 
-https://github.com/heartexlabs/label-studio-frontend
+LSF can be customized and extended to fit your needs, and you can use a custom version of LSF standalone or in your LS installation. For more information see [Custom LSF + LS integration](#custom-lsf-in-label-studio).
+
+LSF is located as a separate GitHub repository: https://github.com/heartexlabs/label-studio-frontend
 
 <br>
-<div style="margin:auto; text-align:center;"><img src="/images/LSF-modules.png" style="opacity: 0.9"/></div>
+<div style="margin:auto; text-align:center;"><img src="/images/frontend/lsf-in-ls.jpg" style="opacity: 0.9"/></div>
```

Comment:
why is the image a .jpg file instead of a .png file?

Reply:
png is good when dealing with transparent background. overall, compression and size-wise jpg is better with all internal optimizations

Reply:
https://heartex.atlassian.net/wiki/spaces/ENGIN/pages/824737793/Label+Studio+Documentation+Handbook#Add-images